noetic/distribution.yaml

```diff
@@ -2458,6 +2458,17 @@ repositories:
       url: https://github.com/ros-visualization/gl_dependency.git
       version: kinetic-devel
     status: maintained
+  gmcl:
+    doc:
+      type: git
+      url: https://github.com/adler-1994/gmcl.git
+      version: noetic-devel
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/adler-1994/gmcl.git
+      version: noetic-devel
```

Comment:
Looks like the `noetic-devel` branch doesn't exist?

Reply:
Hello, I fixed the problem and renamed the brach to noetic-devel. please rerun the test for it.

Reply:
i changed my branch name to master....I hope it will work now flawlessly.

Reply:
Oh, actually I think you had it right the first time you changed it. Sorry if my other comment was confusing.

Reply:
What ever you put here for `version` needs to match the name of the branch in your repository. So, it looks correct at the moment, since you have `master` as the version in this pull request and `master` is the name of your branch in your repository. So it looks ok to me. The failing CI check is due something on our side. I'll fix it as soon as I can.

Hopefully that's clearer. 👍